### ISSUE TITLE
Release 0.19.0 is not compatible with Debian stable

### ISSUE THREAD

**Issue by infertux** [gcash/bchd#504]
[bchd 0.19.0](https://github.com/gcash/bchd/releases/tag/v0.19.0) cannot run on Debian 11 (current stable release):

```
$ ./bchd [...]
bchd: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by bchd)
bchd: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by bchd)
```

[Debian stable ships with glibc 2.31](https://packages.debian.org/search?keywords=libc6&searchon=names&exact=1&suite=all&section=all) but bchd seems to require 2.32+ since this release (bchd 0.18.1 runs fine though).

Would it be possible to recompile it with support for glibc 2.31? Thank you.

/cc @zquestz

**Comment by zquestz** [comment]
Unfortunately I only have access to an arch box, and `gox` which we use for cross compiles is picking up the latest glibc.

I will need to setup a VM with debian and run the cross compile there.

We used to run the cross compiles on travis, but only @cpacia has access to fix that, as I am not an administrator on the repo.

I will see what I can do to get this fixed.

**Comment by zquestz** [comment]
For now, you can easily just compile bchd yourself, that should work just fine.

**Comment by zquestz** [comment]
I just posted a new binary, can you please test it out and see if it works for you?

**Comment by infertux** [comment]
> I just posted a new binary, can you please test it out and see if it works for you?

Thank you for the quick rebuild. I downloaded [the new release](https://github.com/gcash/bchd/releases/download/v0.19.0/bchd-linux_amd64.zip) (new SHA256 checksum: bf6da1a70defaea55baad32fcf129b032945a06baa9cc72363c9c02f7eba264e) and it works fine on Debian.

BTW it would be nice to have the checksums on the release page or some file in the repo.

Thank you for your help. Closing this issue.
